retrieval_service/Dockerfile-cloudsql

```
@@ -0,0 +1,37 @@
+# Copyright 2023 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Set python runtime
+FROM python:3.11-slim
+
+RUN apt-get update \
+  && DEBIAN_FRONTEND=noninteractive \
+  apt-get install --no-install-recommends --assume-yes \
+  postgresql-client
+
+COPY --from=gcr.io/cloud-sql-connectors/cloud-sql-proxy /cloud-sql-proxy /cloudsql/cloud-sql-proxy
+
+# Copy local code to the container image.
+WORKDIR /app
+
+# Install dependencies.
+COPY ./requirements.txt requirements.txt
+COPY ./requirements-test.txt requirements-test.txt
```

nit: I think you can one line this 
```suggestion
COPY ./requirements.txt ./requirements-test.txt ./
```